src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -424,6 +425,14 @@ private function createFirewall(ContainerBuilder $container, string $id, array $
         $container->register('security.user_checker.chain.'.$id, ChainUserChecker::class)
             ->addArgument(new TaggedIteratorArgument('security.user_checker.'.$id));
 
+        // Register Firewall-specific user checker for 'security.helper' login()
+        $securityUserCheckerLocator = $container->getDefinition('security.user_checker_locator');
```

Comment:
i "copied" such things for another locator
please tell me if its the right thing to do